tests/vse_sync_pp/analyzers/test_ppsdpll.py

```diff
@@ -7,31 +7,36 @@
 from decimal import Decimal
 
 from vse_sync_pp.analyzers.ppsdpll import (
-    PhaseOffsetTimeErrorAnalyzer
+    PpsDpllAnalyzer
 )
 
 from .test_analyzer import AnalyzerTestBuilder
 
-DPO = namedtuple('DPO', ('phaseoffset',))
+DPLLS = namedtuple('DPLLS', ('timestamp','eecstate', 'state', 'terror',))
 
-class TestPhaseOffsetTimeErrorAnalyzer(TestCase, metaclass=AnalyzerTestBuilder):
-    """Test cases for vse_sync_pp.analyzers.ppsdpll.PhaseOffsetTimeErrorAnalyzer"""
-    constructor = PhaseOffsetTimeErrorAnalyzer
-    id_ = 'ppsdpll/phase-offset-time-error'
-    parser = 'dpll/phase-offset'
+class TestTimeErrorAnalyzer(TestCase, metaclass=AnalyzerTestBuilder):
```

Comment:
Noting that the name of the unit class no longer relates to the class name in `constructor`, the class under test.